src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagEmailService.kt

```diff
@@ -0,0 +1,88 @@
+ */
+
+package nl.info.zac.productaanvraag
+
+import jakarta.enterprise.context.ApplicationScoped
+import jakarta.inject.Inject
+import net.atos.client.klant.KlantClientService
+import net.atos.client.klant.model.SoortDigitaalAdresEnum
+import net.atos.zac.admin.model.ZaakafhandelParameters
+import net.atos.zac.mailtemplates.MailTemplateService
+import net.atos.zac.mailtemplates.model.MailGegevens
+import nl.info.client.zgw.shared.ZGWApiService
+import nl.info.client.zgw.zrc.model.generated.NatuurlijkPersoonIdentificatie
+import nl.info.client.zgw.zrc.model.generated.OrganisatorischeEenheidIdentificatie
+import nl.info.client.zgw.zrc.model.generated.Zaak
+import nl.info.zac.mail.MailService
+import nl.info.zac.mail.model.MailAdres
+import nl.info.zac.mail.model.getBronnenFromZaak
+import nl.info.zac.productaanvraag.exception.EmailAddressNotFoundException
+import nl.info.zac.productaanvraag.exception.InitiatorNotFoundException
+import nl.info.zac.productaanvraag.exception.MailTemplateNotFoundException
+import nl.info.zac.util.AllOpen
+import nl.info.zac.util.NoArgConstructor
+import nl.info.zac.zaak.ZaakService
+
+@ApplicationScoped
+@NoArgConstructor
+@AllOpen
+class ProductaanvraagEmailService @Inject constructor(
+    private val zgwApiService: ZGWApiService,
+    private val klantClientService: KlantClientService,
+    private val zaakService: ZaakService,
+    private val mailService: MailService,
+    private val mailTemplateService: MailTemplateService,
+) {
+    fun sendEmailForZaakFromProductaanvraag(
+        zaakFromProductaanvraag: Zaak,
+        zaakafhandelParameters: ZaakafhandelParameters
+    ) {
+        zaakafhandelParameters.automaticEmailConfirmation?.takeIf { it.enabled }?.let { automaticEmailConfirmation ->
+            val to = extractInitiatorEmail(zaakFromProductaanvraag)
+            val mailTemplate = mailTemplateService
+                .findMailtemplateByName(automaticEmailConfirmation.templateName)
+                .orElseThrow {
+                    MailTemplateNotFoundException(
+                        "No mail template found with name: '${automaticEmailConfirmation.templateName}'"
+                    )
+                }
+            val mailGegevens = MailGegevens(
+                MailAdres(automaticEmailConfirmation.emailSender, null),
+                MailAdres(to, null),
+                MailAdres(automaticEmailConfirmation.emailReply, null),
+                mailTemplate.onderwerp,
+                mailTemplate.body,
+                null,
+                true
+            )
+            mailService.sendMail(mailGegevens, zaakFromProductaanvraag.getBronnenFromZaak())
+            zaakService.setOntvangstbevestigingVerstuurdIfNotHeropend(zaakFromProductaanvraag)
+        }
+    }
+
+    @Suppress("ThrowsCount")
+    private fun extractInitiatorEmail(createdZaak: Zaak): String {
+        val betrokkene = zgwApiService.findInitiatorRoleForZaak(createdZaak)?.betrokkeneIdentificatie
```

Comment:
```suggestion
        val betrokkeneIdentificatie = zgwApiService.findInitiatorRoleForZaak(createdZaak)?.betrokkeneIdentificatie
```

it is not the betrokkene, it is the betrokkeneIdentificatie.